@xen-orchestra/web-core/lib/locales/en.json

```
@@ -561,6 +561,10 @@
   "skipped": "Skipped",
   "smart-mode": "Smart mode",
   "snapshot": "Snapshot",
+  "snapshot-mode": "Snapshot mode",
+  "snapshot-mode.normal": "Normal",
+  "snapshot-mode.offline": "Offline",
+  "snapshot-mode.with-memory": "With memory",
```

```suggestion
"normal": "Normal",
 "offline": "Offline",
 "with-memory": "With memory",
```

Maybe we could use this keys if we need to reuse ?